OpenSim/Simulation/Model/Appearance.h

```diff
@@ -0,0 +1,151 @@
+#ifndef OPENSIM_APPEARANCE_H_
+#define OPENSIM_APPEARANCE_H_
+/* -------------------------------------------------------------------------- *
+ *                             OpenSim:  Appearance.h                         *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
+ * Author(s): Ayman Habib                                                     *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+
+// INCLUDE
+#include <OpenSim/Simulation/osimSimulationDLL.h>
+#include <OpenSim/Common/Object.h>
+
+namespace OpenSim {
+
+class Body;
+class Model;
+
+
+//=============================================================================
+//=============================================================================
+/**
+ * A class that holds the Display Attributes (Appearance) of an object displayed 
+ * in the OpenSim Visualizer.
+ * 
+ * BaseAppearance contains Appearance properties that apply to all geometry.
+ * Geometry that have a surface so that it can be textured can use the subclass
+ * Appearance, while schematic line drawings (e.g. Arrows, Frames) can use 
+ * LineAppearance which offers thickness. 
+ *
```

Comment:
What is an Appearance? I think we stick to Visualizer (as the abstraction that visualizes Geometry), visualize (the action) and visualization as the noun for the product of a Visualizer then we can get rid of the use of Displayer, display/ed and the display (noun).

Still a fuzzy issue is what is the difference between an Appearance and a Property of a piece of Geometry? When should an attribute be part of the Appearance? For example, thickness. How thick a line could be a Property of a Line itself? Then there would be no need for LineAppearance, would there? Being as precise as we can about what is (and therefore what is not) an Appearance would help a lot. For example, If I create a new Geometry called BlobGeometry and it has a <lumpy> attribute which defines its Geometry and its visualization (similar to a sphere's radius) should I create a LumpyAppearance or BlobAppearance as well or instead? I don't know the answer.